pyro/contrib/gp/kernels/isotropic.py

```diff
@@ -104,9 +104,10 @@ def __init__(self, input_dim, variance=None, lengthscale=None, active_dims=None,
 
 class RationalQuadratic(Isotropy):
     """
-    Implementation of Rational Quadratic kernel: ``(1 + 0.5 * r^2 / alpha l^2)^(-alpha)``.
+    Implementation of RationalQuadratic kernel
+    :math:`\\left(1 + 0.5 \\times \\frac{r^2}{\\alpha l^2}\\right)^{-\\alpha}`.
```

Comment:
nit: It's cleaner to use raw string literals for docstrings with backslashes, e.g. this is what PyTorch does:
```py
r"""
Implementation of ...
:math:`\left(...\alpha}`.
"""
```
Could you use raw docstring literals throughout and remove the double backslashes? (this will help avoid accidental escapes like in `:math:\exp\\left`above :wink:.